cds_migrator_kit/migration_config.py

```diff
@@ -0,0 +1,339 @@
+"""
+InvenioRDM settings for CDS-RDM project.
+
+This file was automatically generated by 'invenio-cli init'.
+
+For the full list of settings and their values, see
+https://inveniordm.docs.cern.ch/reference/configuration/.
+"""
+
+import os
+
+from datetime import datetime, timedelta
```

Comment:
this is a copy of our invenio.cfg

Do you maybe have better ideas on how to tackle this problem of duplication?

Reply:
Let's discuss IRL to be sure we understand well the needs.